programmers/수박수박수박수박수박수.js

```diff
@@ -0,0 +1,9 @@
+function solution(n) {
+  let answer = "";
+  let plusW = "";
+  for (let i = 0; i < n; i++) {
+    plusW = i % 2 == 0 ? "수" : "박";
+    answer += plusW;
```

Comment:
```suggestion
    (i % 2 === 0) ? answer += "수" : answer += "박";
```
굳이 plusW라는 변수를 선언하지 않아도 될 것 같네요.

그리고 어제 말씀드린 것처럼 == 보다는 ===의 사용을 권장드립니다.